diff --git a/android/build.gradle b/android/build.gradle
index e87e6eaf1d3da0f84d2324f336b96f5e647fc747..3230ea11404a89315fb6cb59a7c5fd473ba62508 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -13,7 +13,7 @@ buildscript {
 
     dependencies {
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
-        classpath("com.android.tools.build:gradle:7.2.1")
+        classpath("com.android.tools.build:gradle:8.1.1")
         classpath("com.diffplug.spotless:spotless-plugin-gradle:6.7.2")
     }
 }
